Userland/Applications/3DFileViewer/main.cpp

```diff
@@ -184,6 +190,58 @@ int main(int argc, char** argv)
         app->quit();
     }));
 
+    auto& view_menu = menubar->add_menu("&View");
+    view_menu.add_action(GUI::CommonActions::make_fullscreen_action([&](auto&) {
+        window->set_fullscreen(!window->is_fullscreen());
+    }));
+
+    auto& rotation_axis_menu = view_menu.add_submenu("Rotation Axis");
+    auto rotation_x_action = GUI::Action::create_checkable("x", [&widget]() {
```

Comment:
Please omit `()` for lambdas with no parameters. Same everywhere below.

```suggestion
    auto rotation_x_action = GUI::Action::create_checkable("&X", [&widget] {
```

Reply:
Just taking a look at the file, I could also remove the `()` around the lambdas on lines 187 and 192, would you like me to? Not sure if that sort of change would be considered outside the scope of this PR :)